From cbf6c57678967b996e66edcf9330aee44068235f Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Wed, 31 Jul 2019 00:03:07 +0200
Subject: [PATCH] Fix -fPIE casing

-fpie seems to work, but bhlc complains that -fPIE is missing

Bug: https://bugzilla.samba.org/show_bug.cgi?id=14061
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: cifs-utils/configure.ac
===================================================================
--- cifs-utils.orig/configure.ac
+++ cifs-utils/configure.ac
@@ -130,7 +130,7 @@ AC_CHECK_FUNC(setfsuid, , [AC_MSG_ERROR(
 
 # FIXME: add test(s) to autodisable these flags when compiler/linker don't support it
 if test $enable_pie != "no"; then
-	PIE_CFLAGS="-fpie -pie"
+	PIE_CFLAGS="-fPIE -pie"
 else
 	PIE_CFLAGS=""
 fi
